tests/utilities/fake_driver.py

```diff
@@ -2,13 +2,13 @@
 from __future__ import annotations
 
 import sys
-from enum import Enum, IntEnum
+from enum import Enum
 from types import TracebackType
 from typing import TYPE_CHECKING, Any, ContextManager, Dict, Optional, Type
 
-if TYPE_CHECKING:
-    import grpc
+from ni_measurementlink_service.session_management._types import SessionInitializationBehavior
```

Comment:
Driver APIs do not depend on ni_measurementlink_service.

Driver APIs define their own SessionInitializationBehavior enums.

The MeasurementLink specific glue for each driver API maps the MeasurementLink enum to the driver enum, but this doesn't belong in the driver API.